packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -325,12 +335,18 @@ class ThemeData {
   /// A text theme that contrasts with the primary color.
   final TextTheme primaryTextTheme;
 
+  /// A text theme that contrasts with the accent color.
+  final TextTheme accentTextTheme;
+
   /// An icon theme that contrasts with the card and canvas colors.
   final IconThemeData iconTheme;
 
   /// An icon theme that contrasts with the primary color.
   final IconThemeData primaryIconTheme;
 
+  /// An icon theme that contrasts with the accent color.
+  final IconThemeData accentIconTheme;
+
   /// The platform the material widgets should adapt to target.
   ///
   /// Defaults to the current platform.
```

Comment:
I now have `accentTextTheme`s and `accentIconTheme`s in all my various constructors and methods.